maintainers/maintainer-list.nix

```diff
@@ -11535,6 +11535,13 @@
     githubId = 2666479;
     name = "Y Nguyen";
   };
+  superherointj = {
```

Comment:
Do you want to revert https://github.com/NixOS/nixpkgs/pull/144319?

Reply:
No. Thanks for asking.